src/select-menu/docs/examples/SelectMenu-multi.example

```diff
@@ -0,0 +1,46 @@
+<Manager>
+       {({ setState, state }) => (
+              <SelectMenu
+                title="Select name"
+                options={options}
+                selected={state.selected}
+                onSelect={item => {
+            let selected
+            if (Array.isArray(state.selected)) {
+              selected = [...state.selected, item.value]
+            } else {
+              selected = arrify(item.value)
+            }
+            setState({
+              selected,
+              selectedNames: selected.toString().substr(0, 25) + '...'
+            })
+          }}
+          onDeselect={item => {
+            const DeselectedItemIndex = state.selected.indexOf(item.value)
+            const selectedItems = state.selected.filter(
+              (_item, i) => i !== DeselectedItemIndex
+            )
+
+            const selectedItemsLength = selectedItems.length
+            let selectedNames = ''
+            switch (true) {
```

Comment:
Wow, I just learned something new in JS. Never thought you could do this with a switch case. We should probably dumb it down to use if statements instead, I think this might not familiar for a lot of people (including me haha).

Reply:
Ahh makes sense! I started with C++ via Arduino a couple years back actually, so this is partly that shining through. Good deal.